### ISSUE TITLE
Web Notify not working in POS

### ISSUE THREAD

**Issue by dpavlic** [OCA/web#2398]
## Module
web_notify

## Describe the bug
I'm actually not certain if this is a bug or not, but web notifications do not show in the point of sale.

## To Reproduce
**Affected versions**: Latest

1. Install the module
2. Try and run the notification trigger within the POS

**Expected behavior**
Notification should work in the POS same as in any other part of the website.

**Additional context**
Latest web_notify and Python, Odoo 15

**Comment by legalsylvain** [comment]
Hi. That is a know issue. (Maybe not documented, could you make a PR ?)
It requires a non existing module. pos_notify. To do.

Regards.

**Comment by dpavlic** [comment]
Any idea why it doesn't work? I'd love to give this a go since I'm personally invested into a fix, but am a newbie to Odoo. I've spent some time in understanding Odoo's data model and the Owl component architecture but don't really understand how the bus works for the Python -> JS interop. 

I tried to make an Owl POS component which would listen to the bus for the web_notify event and console log the result, but nothing is ever returned. I'm gathering the whole setup works a bit differently in the POS view: wondering if perhaps you have some wisdom to share here?
